.github/workflows/docker-coq.yml

```
@@ -98,3 +98,26 @@ jobs:
         custom_script: etc/ci/github-actions-docker-make.sh TIMED=1 validate COQCHKFLAGS="-o ${COQCHKEXTRAFLAGS}"
       if: env.SKIP_VALIDATE == '' && github.event_name != 'pull_request'
 
+  test-amd64:
+
+    runs-on: ubuntu-latest
+
+    concurrency:
+      group: ${{ github.workflow }}-test-amd64-${{ github.head_ref || github.run_id }}
+      cancel-in-progress: true
+    needs: docker-build
+
+    steps:
+    - name: checkout repo
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: Download a Build Artifact
+      uses: actions/download-artifact@v3
+      with:
+        name: ExtractionOCaml-master
+        path: src/ExtractionOCaml
+    - name: make binaries executable
+      run: chmod +x src/ExtractionOCaml/solinas_reduction src/ExtractionOCaml/word_by_word_montgomery src/ExtractionOCaml/unsaturated_solinas src/ExtractionOCaml/dettman_multiplication
```

Might be more future proof to do
```suggestion
      run: chmod +x src/ExtractionOCaml/*
```
?
(Also, do you need `sudo`? And/or to say `chmod a+x`? Or `chown`?)